docs/src/release-notes-js.md

```diff
@@ -8,57 +8,57 @@ import LiteYouTube from '@site/src/components/LiteYouTube';
 
-Playwright now also ships `chromium-headless-shell` channel that is a separate build that closely follows the old headless implementation. If you would like to keep the old behavior before you are ready to switch to the new headless, please fallback to this channel:
+#### Chromium headless shell
 
-1. First, install this channel prior to running tests. Make sure to list all browsers that you use.
+Starting with this release, Playwright downloads and runs two different browser builds - one is a regular headed chromium and the other is a chromium headless shell. This should be transparent to you, **no action is needed**. You can learn more in [issue #33566](https://github.com/microsoft/playwright/issues/33566).
 
-  ```bash
-  # running tests in all three browsers, headless and headed
-  npx playwright install chromium chromium-headless-shell firefox webkit
+If you are only running tests in headless, for example on CI, you can avoid downloading a headed version of Chromium by specifying `chromium-headless-shell` during installation.
 
-  # running tests in all three browsers on CI, headless only
-  npx playwright install chromium-headless-shell firefox webkit
-  ```
+```bash
+# only running tests headlessly
+npx playwright install chromium-headless-shell firefox webkit
+```
 
-1. Update your config file to specify `'chromium-headless-shell'` channel.
+Playwright will skip downloading headed chromium build, and will use `chromium-headless-shell` when running headless.
 
-  ```js
-  import { defineConfig, devices } from '@playwright/test';
+#### Opt-in to new headless
 
-  export default defineConfig({
-    projects: [
-      {
-        name: 'chromium',
-        use: {
-          ...devices['Desktop Chrome'],
-          channel: 'chromium-headless-shell',
-        },
-      },
-      {
-        name: 'firefox',
-        use: { ...devices['Desktop Firefox'] },
-      },
-      {
-        name: 'webkit',
-        use: { ...devices['Desktop Safari'] },
-      },
-    ],
-  });
-  ```
+We encourage everyone to try and switch to the new headless by using the `chromium-next` channel.
+
+First, install this channel prior to running tests. Make sure to list all the browsers that you use.
+
+```bash
+npx playwright install chromium-next firefox webkit
+```
 
-1. Note that `chromium-headless-shell` channel only supports headless operations. If you try to run tests in headed mode, it will automatically fallback to regular `chromium`.
+Then update your config file to specify `'chromium-next'` channel.
+
+```js
+import { defineConfig, devices } from '@playwright/test';
+export default defineConfig({
+  projects: [
+    {
+      name: 'chromium',
+      use: {
+        ...devices['Desktop Chrome'],
+        channel: 'chromium-next',
```

Comment:
Should we put:
- chromium-next
- download only needed browsers (I think a great one)
In our best practices guide?

Reply:
Feel free to make a PR 😄